### DISCUSSION TITLE
Allow Bookmarks toolbar to be above tabs

### DISCUSSION THREAD

**Discussion by maverick74** [Floorp-Projects/Floorp#730] [Feature Request]
Bookmarks toolbar does not make any sense to live bellow the URL bar.

 Bookmarks are basically shortcuts and has nothing to do with the page we're at, so why does it live bellow the URL bar?

Having it in this order:

Bookmarks bar
Tabs bar
URL bar

makes more sense, iyam.

hence, my suggestion!

**Comment by Telmesomething** [comment]
As an alternative, right click a blank part of the tab bar and checkmark Menu bar. You now have a Bookmarks link above the URL bar. If you don't like it up there permanently, just hit the Alt key when you want to access it.

**Reply by maverick74** [reply]
Yes, I've tried that before! 
But I can't remove or hide the menu entries (it self) in order to use the complete toolbar space!

I end up having the menu bar entries - which I do not use - and the rest of the space for "half" bookmarks bar...


hence, this feature request

**Reply by Telmesomething** [reply]
Well, you could conceivably place the URL bar at the foot of the browser and move the tab bar down there as well which leaves only the Bookmarks toolbar at the top. 

Personally, I use the left hand sidebar for bookmarks (_I don't like the tree style option_) and  the menu bar is permanently enabled at the top. I need it there because I don't like those javascript menus which are located on the right hand side.

**Reply by maverick74** [reply]
i don't like Toolbars at the bottom and to my daily work it's really useful having the bookmarks bar.

But still... i like to keep the browser out of sight as much as possible (within usefulness, of course! Complete fullscreen is not that useful...). Menu bar is not essential to me (in fact, i can totally live without it)!

Naturally this is not a "must have" feature. It's more just as "would be nice to have"!

(i really don't get it why mozilla locked URL and Menu on it's place. If a user want's to move things around, why don't they just allow it... but then again... i'm more of a _KDE_ guy than a _Gnome_ guy... i like being able to do extreme personalizations eheheh)
